### ISSUE TITLE
Add URL Filtering rules (Black List) to ECF Assembly

### ISSUE THREAD

**Issue by bruce-wh-li** [bcgov/cloud-pathfinder#2076]
**Describe the issue**
Block ingress traffic to firewall and firewall manager from internet
Enable URL Filtering in FW in AWS Cloud
Retrofit ECF Forge 

**Additional context**

- Duplicate the success from Forge to block ingress traffic to firewall and firewall manager from internet
- Add firewall rules to deny Traffic to/from the websites of bad url categorization. 
- Only applicable to traffic to VPC where FW exists
https://urlcat.checkpoint.com/urlcat/categories.htm

**Acceptance Criteria**
- Contact Prabhu for help accessing the Firewall configs
- Following Forge example to block ingress traffic to firewall and firewall manager from internet
- Identify the list of bad categorizations to be blocked (black listed)
e.g. Anonymizer, Botnets, Critical Risk, Hacking, High Risk, Medium Risk, P2P File Sharing, Phishing, Spam, Spyware/Malicious Sites, Suspicious Content, Web Advertisements. 
- Run a little mural activity in the Community Update and feed the results into this ticket
- Announce in RC channel in prior to deployment
- Add rule to Deny egress traffic to website of bad url categories (drop and log)
- Curl command to check connection denied or time out
- Check firewall log for evidence it is blocked by the FW rule added
- Hand over these test steps to Carles so that Carles can include in a separate ticket for adding to the test stack

**Comment by bruce-wh-li** [comment]
Firewall and Firewall Manager does not communicate.  ASG Health Check is not Healthy.   
Tech Team and CheckPoint are currently working on it.   Suggest wait until the problem resolved and firewall stabilized before adding new firewall rules.

**Comment by lukegonis** [comment]
Based on client feedback and other considerations, the team decided not to proceed with this change.
